advisories/github-reviewed/2023/11/GHSA-8r96-8889-qg2x/GHSA-8r96-8889-qg2x.json

```diff
@@ -32,11 +28,14 @@
               "introduced": "0"
             },
             {
-              "last_affected": "3.2.2"
+              "fixed": "3.2.3"
```

Comment:
This is the only change I actually made - setting the "fixed" version - the rest of the changes seem to have been made automatically.